client/gutenberg/extensions/related-posts/style.scss

```diff
@@ -0,0 +1,34 @@
+.wp-block-a8c-related-posts {
+	[data-align='full'] & {
+		padding: 0 20px;
+	}
+}
+
+.wp-block-a8c-related-posts__preview-items {
+	&.is-grid {
+		display: flex;
+		padding: 0 10px;
+		margin: 0 -20px;
+
+		.wp-block-a8c-related-posts__preview-post {
+			margin: 0 10px;
+		}
+	}
+
+	&.columns-3 .wp-block-a8c-related-posts__preview-post {
+		width: 33.33%;
```

Comment:
Good point 👍 I drafted this CSS very quickly when I was building this block, and it seems I haven't updated it since.

I've updated to use flex now in https://github.com/Automattic/wp-calypso/pull/26530/commits/d0c4c91a5a25cd89cc1520ea54bd2b4801846bf7

We can simply use `flex-grow` and `flex-basis` and we won't need to specify `min-width` at all.